homeassistant/components/mopeka/strings.json

```
@@ -18,5 +18,15 @@
       "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "options": {
+    "step": {
+      "init": {
+        "title": "Configure Mopeka",
+        "data": {
+          "medium_type": "Medium Type"
```

```suggestion
          "medium_type": "[%key:component::mopeka::config::step::user::data::medium_type%]"
```

Once https://github.com/home-assistant/core/pull/122705/files#r1696052719 is addressed, you can do it by reference so the translation team only has to translate it once